patches/tock/01-persistent-storage.patch

```diff
@@ -18,9 +18,9 @@ index ddac9dbd..7e2a3298 100644
              kernel::ipc::DRIVER_NUM => f(Some(&self.ipc)),
              _ => f(None),
          }
-@@ -408,6 +410,14 @@ pub unsafe fn setup_board<I: nrf52::interrupt_service::InterruptService>(
-         None
-     };
+@@ -405,6 +407,14 @@ pub unsafe fn setup_board<I: nrf52::interrupt_service::InterruptService>(
+     );
+     nrf52::acomp::ACOMP.set_client(analog_comparator);
```

Comment:
This line surely has nothing to do with persistent storage.
Isn't this line supposed to be added anyway by bumping the commit on `third_party/tock`?